python/solutii/vlad_cristia_avram/unic.py

```diff
@@ -0,0 +1,17 @@
+def gaseste_unic(istoric):
+    for i in istoric[1:]:
```

Comment:
Folosești memorie suplimentară.
```
istoric[1:]
``` 
va crea o nouă listă ce conține toatele elementele din istoric mai puțin primul.